bin/README.md

```diff
@@ -47,9 +47,41 @@ $ kuzzle createFirstAdmin
 ```
 
 will guide you through the creation process of the first admin user and fix the rights to other user types if needed.
+<<<<<<< HEAD
 
 **Note:** This command is interactive and let you choose to reset the roles rights or not.
 
+# Reset Kuzzle
+
+```
+$ kuzzle likeAvirgin
+```
+
+will allow you to reset Kuzzle and restore it as if it is freshly installed.
```

Comment:
To make it clearer, I would delete the note below, and include a full description of what this command does.

Something like this:
"Interactive command resetting Kuzzle data, and eventually Kuzzle rights"

What do you think?